userguide/tutorials/stripe_plugin.adoc

```diff
@@ -0,0 +1,230 @@
+----
+$ git clone https://github.com/killbill/killbill-stripe-demo.git
+----
+
+Next, use RVM to install Ruby (Ruby 2.1+ or JRuby 1.7.20+ is recommended).
+
+[source, bash]
+----
+$ gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+\curl -sSL https://get.rvm.io | bash -s stable --ruby
+----
+
+Afterwards, install the dependencies by running in the parent folder:
+
+[source, bash]
+----
+$ gem install bundler
+$ bundle install
+----
+
+Obtain the IP address of your Kill Bill Docker container by running the following:
+
+[source,nash]
+----
+$ docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <CONTAINER_ID>
+----
+
+Now, you can run the app using your Stripe publishable key:
+
+[source,bash]
+----
+$ KB_URL='http://<KILL_BILL_CONTAINER_IP>:8080' PUBLISHABLE_KEY=<YOUR_PUBLISHABLE_STRIPE_KEY> ruby app.rb
+----
+
+In your browser, navigate to: http://localhost:4567/ and click the 'Buy' button.
+
+Enter dummy data (4242 4242 4242 4242 as the credit card number, any three digit CVC and any expiry date in the future work) and complete the checkout process.
+
+=== Step 9:  What just happened?
+
+After we complete the checkout process with Stripe, the card has been tokenized, or intercepted and replaced with a surrogate token ID.  If you visit to the Stripe Dashboard (https://dashboard.stripe.com), you should be able to navigate to the Payments page in the left-hand panel to see a succeeded Kill Bill charge for $10.
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_tokenization.png[align=center]
+
+In Kaui, navigate to the `+/accounts+` page to see the newly created Kill Bill account, and select it:
+
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_kbaccount.png[align=center]
+
+You will see that a new payment method has been associated with this account by expanding 'Payment Methods' in the lower left-hand corner.  A Customer object is also created in Stripe (see `+customer_id+`), so the token associated with this account can be re-used.
+
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_payment_methods.png[align=center]
+
+If you select 'Subscriptions' at the top of the page, you will see our newly creeated John Doe account now has a monthly subscription for our 'Sports' product, at a value of $10.00.
+
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_subscription_bundles.png[align=center]
+
+Navigating to the 'Payments' tab at the top of the page, you will also see that a payment was processed with an AUTH AMOUNT of $10.00.
+
+
+image:https://raw.githubusercontent.com/annabaker/killbill-docs/v3/userguide/assets/img/tutorials/stripe_integration_payments.png[align=center]
+
+
+=== Step 10:  Conclusion
```

Comment:
One idea would be to provide links for next steps:

* API reference (https://killbill.github.io/slate/)
* Production deployment (https://docs.killbill.io/latest/aws.html)
* Community forum for help (https://groups.google.com/forum/#!forum/killbilling-users)